_rules/first-focusable-element-is-link-to-non-repeated-content-8a213c.md

```diff
@@ -0,0 +1,658 @@
+---
+id: 8a213c
+name: First focusable element is link to non-repeated content
```

Comment:
I don't quite understand what the purpose of this rule is. Seems to me that your rule "Document has an instrument to move focus to non-repeated content" does this too, except a little better because it can handle multiple repeating blocks on the same page.